src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Drawable;
+import core.basesyntax.service.FigureSupplier;
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int arrayLength = 10;
```

Comment:
All magic numbers in your code should be constants. Consider defining '10' as a named constant to improve code readability and maintainability.